src/cpp/rtps/builtin/discovery/endpoint/EDPServer.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+// Copyright 2020 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Copyright in modified files stays the shame
```suggestion
// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```